### ISSUE TITLE
Statistics Filtering

### ISSUE THREAD

**Issue by BenWoodworth** [BtoBastian/bStats#50]
I think it would be useful if statistics could be filtered based on other statistics. For example, filtering based on Minecraft version, it would be possible to see the distribution of certain statistics for a given Minecraft version, such as plugin versions or Java versions.

I'm not sure how feasible this would be to implement, but it would certainly provide some more useful insights into how plugins are being used.

**Comment by BtoBastian** [comment]
I agree, that it would be very cool if you could filter the data. However, this would require the data to be stored in a way more complex way which comes with two problems:
1. It would require **a lot** more memory to store the data and CPU power to manage it. The current solution is very performant, allowing me to run the website with very little hardware requirements. This feature could easily double the required resources.
2. I would have to rewrite a lot of the current code and come up with a new database layout -> This would take **a lot** of time.

There's stuff which has a higher priority (e.g. signature images) which is way easier to implement, so I don't think I'm going to implement it in the near future or maybe never. It's definitely possible, but to be honest it's more work than I'm willing to put into this feature.
